PHP/PHP Source.sublime-syntax

```diff
@@ -1261,10 +1261,14 @@ contexts:
         - match: \)
           scope: punctuation.section.group.end.php
           set: after-function-call
-        - match: ','
-          scope: punctuation.separator.php
-        - include: function-call-named-parameters
-        - include: nested-expressions
+        - match: (?=\S)
+          push:
+            - match: (?=\))
+              pop: 1
+            - match: ','
+              scope: punctuation.separator.php
+            - include: function-call-named-parameters
+            - include: nested-expressions
```

Comment:
What if `?>` appears in front of closing `)`? By just pushing into another context you break `nested-expressions`'s bailout, which is implemented in `expression-end-pop` context. The new context is popped by it, if e.g. `;`, `?>` or a keyword is matched but it is immediately pushed on stack again.

I don't think that's a suitable solution.

Just replacing `nested-expressions` by `expressions` would have the same effect but without the pop-push-loop which is just prevented by ST's fail safe mechanisms.

Shouldn't closures correctly push `function-body` onto stack to prevent `expression-end-pop` from matching `;` or `}`?

Reply:
I think we still go the `nested-expressions` but something has to be change in `keywords` variable. Do you think we need to rename it or just let it be.

Reply:
The variable is only used to pop nested expressions at the moment. Not even sure whether all those different keyword types need to be placed in dedicated variables. I'd change it once those are to be used in other situations as well.